packages/cx-editors/doc/RichText.md

```diff
@@ -1,20 +1,26 @@
-# CanvasX Rich Text Editor Component
+# CX Rich Text Editor Component
 
-The CanvasX Rich Text Editor extend the [Bodiless Rich Text](../../bodiless/Components/RichText) and provides default
-editors to be used on site.  By default there are three options for the Rich Text Editor: Simple, Basic, and
-Full.
+The CX Rich Text Editor Component is based on the [BodilessJS Rich Text Editor
+Component](/Components/Editors/RichText). While Bodiless Rich Text is a generic rich text editor
+(RTE) component with tokens that can be combined however you choose, CX Rich Text builds upon it,
+providing a sensible default combination of its generic tokens (i.e., features and styles), to help
+meet typical site-use expectations.
+
+By default, there are three options for the Rich Text Editor: _Simple_, _Basic_, and _Full_.
 
 ## Content Editor Details
 
-There is no change to editor experience by CanvasX Editor package and thus can refer to [Bodiless Rich Text: Content Editor](../../bodiless/Components/RichText?id=content-editor-details)
+There is no change to the Editor experience by the CX Editor package, and, thus, you can refer to
```

Comment:
maybe mention other than "potentially you might see different buttons available"